CHANGELOG.rst

```diff
@@ -1,6 +1,6 @@
 0.14.0
```

Comment:
In many pytest projects we use this format instead:

```rst
0.14.0 (UNRELEASED)
-------------------
```

And then:

```rst
0.14.0 (2020-09-24)
-------------------
```

I think it would be nice to be consistent here, what do you think?

Reply:
Agreed. Let's stick to the convention.